src/Apps/About/AboutApp.tsx

```diff
@@ -84,6 +97,162 @@ export const AboutApp: React.FC<React.PropsWithChildren<unknown>> = () => {
       <Spacer y={6} />
 
       <AboutArtworksRailQueryRenderer />
+
+      <StructuredData
+        schemaData={{
+          "@context": "https://schema.org",
+          "@type": "WebApplication",
+          name: "Artsy",
+          url: "https://www.artsy.net",
+          applicationCategory: "LifestyleApplication",
```

Comment:
Wondering about `ShoppingApplication` from Google's [list of supported values](https://developers.google.com/search/docs/appearance/structured-data/software-app#:~:text=List%20of%20supported%20app%20types). (No idea what substantive difference this makes on the search results page)